include/ofx-props.yml

```diff
@@ -0,0 +1,1508 @@
+########################################################################
```

Comment:
Is this auto generated? If so maybe add it into the header

Reply:
`ofx-props.yml` is hand-created -- it's now the source of truth for all info about properties and property sets. That's the lion's share of the work in this PR (and probably the source of remaining bugs, where I may have gotten some stuff wrong). None of this info was machine-readable before, so I extracted it from the docs and header file comments.